examples/chapter-list.lua

```diff
@@ -20,32 +41,58 @@ local function open_chapter()
     end
 end
 
---dynamic keybinds to bind when the list is open
-list.keybinds = {
-    {'DOWN', 'scroll_down', function() list:scroll_down() end, {repeatable = true}},
-    {'UP', 'scroll_up', function() list:scroll_up() end, {repeatable = true}},
-    {'ENTER', 'open_chapter', open_chapter, {} },
-    {'ESC', 'close_browser', function() list:close() end, {}}
-}
-
 --update the list when the current chapter changes
 mp.observe_property('chapter', 'number', function(_, curr_chapter)
     list.list = {}
     local chapter_list = mp.get_property_native('chapter-list', {})
     for i = 1, #chapter_list do
         local item = {}
-        if (i-1 == curr_chapter) then
+        if (i - 1 == curr_chapter) then
+            list.selected = curr_chapter + 1
             item.style = [[{\c&H33ff66&}]]
         end
```

Comment:
```suggestion
```

Instead of burying this in the loop we can place it somewhere more prominent at the end.